contrib/runners/python_runner/python_runner.py

```diff
@@ -128,8 +129,13 @@ def run(self, action_parameters):
         LOG.debug('Setting env.')
         env = os.environ.copy()
         env['PATH'] = get_sandbox_path(virtualenv_path=virtualenv_path)
-        env['PYTHONPATH'] = get_sandbox_python_path(inherit_from_parent=True,
-                                                    inherit_parent_virtualenv=True)
+
+
+        sandbox_python_path = get_sandbox_python_path(inherit_from_parent=True,
+                                                      inherit_parent_virtualenv=True)
+        pack_common_libs_path = get_pack_common_libs_path(pack_db=pack_db)
+
+        env['PYTHONPATH'] = pack_common_libs_path + ':' + sandbox_python_path
```

Comment:
Just thinking out loud - is adding it to the front of the python path the right thing to do? :)